Murcielago/Murcielago/UserInterface/Localizable.xcstrings

```diff
@@ -1,6 +1,54 @@
 {
   "sourceLanguage" : "en",
   "strings" : {
+    "Connection Status:" : {
+      "localizations" : {
+        "es-419" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Estado de conexión"
+          }
+        },
+        "es-US" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Estado de conexión"
+          }
+        }
+      }
+    },
+    "Controller Settings" : {
+      "localizations" : {
+        "es-419" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Configuración del control"
+          }
+        },
+        "es-US" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Ajustes del control"
+          }
+        }
+      }
+    },
+    "Left Trigger" : {
+      "localizations" : {
+        "es-419" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Gatillo izquierdo"
+          }
+        },
+        "es-US" : {
```

Comment:
If the strings are the same, are both required? Ideally iOS should know to use `es-419` if `es-US` isn't specified?

Reply:
It looks like it should fallback? https://developer.apple.com/library/archive/technotes/tn2418/_index.html

Reply:
This is good to know, but I will be continuing to keep each Spanish variation at 100% translation because I think I will consolidate to only one (just not sure which yet)